pkg/quota/api/types.go

```diff
@@ -115,6 +116,26 @@ func (o *ResourceQuotasStatusByNamespace) OrderedKeys() *list.List {
 	return o.orderedMap.OrderedKeys()
 }
 
+func init() {
```

Comment:
now that we were actually copying the internal fields, semantic deepequal choked on comparing different unexported fields. we need to tell it how to compare our custom type. not sure where to hook this (AddKnownTypes? init() here?)